desktop/cmp/input/Select.js

```diff
@@ -515,6 +516,18 @@ export class Select extends HoistInput {
             () => Icon.chevronDown({className: 'xh-select__indicator'});
     }
 
+    // As per example @ https://react-select.com/components#replaceable-components
```

Comment:
just curious about the asymmetry with our custom dropdown indicator above.    Are we potentially missing something there with respect to refs, and additional props?

Reply:
I had the same question - I can tell you the clear indicator required the innerProps to be spread on the div as that's what wired up the correct / provided click handler.